tmt/steps/report/reportportal.py

```diff
@@ -14,85 +14,124 @@
 
 @dataclasses.dataclass
 class ReportReportPortalData(tmt.steps.report.ReportStepData):
+
+    @staticmethod
+    def _flag_env_to_default(option: str, default: bool) -> bool:
+        env_var = 'TMT_PLUGIN_REPORT_REPORTPORTAL_' + option.upper()
+        if env_var not in os.environ:
+            return default
+        return bool(os.getenv(env_var) == '1')
+
+    @staticmethod
+    def _str_env_to_default(option: str, default: Optional[str]) -> Optional[str]:
+        env_var = 'TMT_PLUGIN_REPORT_REPORTPORTAL_' + option.upper()
+        if env_var not in os.environ or os.getenv(env_var) is None:
+            return default
+        return str(os.getenv(env_var))
+
     url: Optional[str] = field(
         option="--url",
         metavar="URL",
-        default=os.getenv('TMT_PLUGIN_REPORT_REPORTPORTAL_URL'),
+        default=_str_env_to_default('url', None),
         help="The URL of the ReportPortal instance where the data should be sent to.")
+
     token: Optional[str] = field(
         option="--token",
         metavar="TOKEN",
-        default=os.getenv('TMT_PLUGIN_REPORT_REPORTPORTAL_TOKEN'),
+        default=_str_env_to_default('token', None),
         help="The token to use for upload to the ReportPortal instance (from the user profile).")
+
     project: Optional[str] = field(
         option="--project",
         metavar="PROJECT_NAME",
-        default=os.getenv('TMT_PLUGIN_REPORT_REPORTPORTAL_PROJECT'),
+        default=_str_env_to_default('project', None),
         help="Name of the project into which the results should be uploaded.")
+
     launch: Optional[str] = field(
         option="--launch",
         metavar="LAUNCH_NAME",
-        default=os.getenv('TMT_PLUGIN_REPORT_REPORTPORTAL_LAUNCH'),
-        help="The launch name (name of plan per launch is used by default).")
+        default=_str_env_to_default('launch', None),
+        help="Set the launch name, otherwise name of the plan is used by default. Should be"
+             " defined with suite-per-plan option or it will be named after the first plan")
```

Comment:
`help` can be a multi-line string, which is better than multiple joined strings (no easy-to-forget leading/trailing spaces):

```python
        help="""
             Set the launch name, otherwise name of the plan is used by default.
             Should be defined with suite-per-plan option or it will be named
             after the first plan
             """)
```